content/en/blog/gke-firestore/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Guide to have GKE based application use Firestore as database"
+description: ""
+excerpt: "Guide to have GKE based application use Firestore as database"
+date: 2024-08-23T09:19:42+01:00
+lastmod: 2024-08-23T09:19:42+01:00
+draft: false
+images: []
+categories: ["Kubernetes", "Firestore", "Workload-identity"]
+tags: ["Kubernetes", "Firestore", "Workload-identity"]
+contributors: ["balabtech"]
+pinned: true
+homepage: true
+---
```

Comment:
suggestion to add weight:42 so that the blog will appear as latest and an image too, @Balasundaram
rest looks good!

Reply:
in future commits, we make the weight value same for all blogs, so that it will be ordered as per the timestamp of modification.

Reply:
@Balasundaram contributors: do you want to switch to a different ID?.

Reply:
Changes are made for weight and image. For my Id Sathish, i think balasundaram is good :)

Reply:
CHanges are added. To my id, Balasundaram looks good :)